frontend/src/metabase/visualizations/components/ObjectDetail/ObjectDetail.unit.spec.tsx

```diff
@@ -13,17 +14,17 @@ import {
 } from "./ObjectDetail";
 import type { ObjectDetailProps } from "./types";
 
+const mockQuestion = new Question(
+  createMockCard({
+    name: "Product",
+  }),
+);
+
 function setup(options?: Partial<ObjectDetailProps>) {
-  render(
+  renderWithProviders(
```

Comment:
These changes are needed because of `useActionListQuery` and `useDatabaseListQuery` usage in `ObjectDetail`.

I didn't want to use deprecated HOCs and I didn't want to introduce another wrapper component for `ObjectDetail` - this is the consequence.

Please let me know if something else is preferable.